src/components/NavBar/NavBar.jsx

```diff
@@ -16,7 +16,7 @@ import Toolbar from '@mui/material/Toolbar';
 import Typography from '@mui/material/Typography';
 import { useTheme } from '@mui/material/styles';
 // Component Imports
-import LogoutModal from '../LogoutModal/LogoutModal';
+import Modal from '../LogoutModal/LogoutModal';
```

Comment:
all of this goes back to my comments about changing `handleDeleteClient` to just `handleDelete`, same concept.

just changing it all to be less specific.